src/views/AccountInfo/AccountInfoSection.tsx

```diff
@@ -0,0 +1,343 @@
+import { shallowEqual } from 'react-redux';
+import styled, { css } from 'styled-components';
+
+import type { Nullable, TradeState } from '@/constants/abacus';
+import { ButtonAction, ButtonShape, ButtonSize, ButtonStyle } from '@/constants/buttons';
+import { ComplianceStates } from '@/constants/compliance';
+import { DialogTypes } from '@/constants/dialogs';
+import { STRING_KEYS } from '@/constants/localization';
+import { DydxChainAsset } from '@/constants/wallets';
+
+import { useAccounts } from '@/hooks/useAccounts';
+import { useBreakpoints } from '@/hooks/useBreakpoints';
+import { useComplianceState } from '@/hooks/useComplianceState';
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import breakpoints from '@/styles/breakpoints';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Button } from '@/components/Button';
+import { Details } from '@/components/Details';
+import { Icon, IconName } from '@/components/Icon';
+import { IconButton } from '@/components/IconButton';
+import { MarginUsageRing } from '@/components/MarginUsageRing';
+import { OutputType } from '@/components/Output';
+import { WithSeparators } from '@/components/Separator';
+import { WithTooltip } from '@/components/WithTooltip';
+
+import { calculateIsAccountLoading } from '@/state/accountCalculators';
+import { getSubaccount } from '@/state/accountSelectors';
+import { useAppDispatch, useAppSelector } from '@/state/appTypes';
+import { openDialog } from '@/state/dialogs';
+
+import { isNumber, MustBigNumber } from '@/lib/numbers';
+import { testFlags } from '@/lib/testFlags';
+
+import { AccountInfoDiffOutput } from './AccountInfoDiffOutput';
+
+enum AccountInfoItem {
+  PortfolioValue = 'portfolio-value',
+  MarginUsed = 'margin-used',
+
+  // TODO: CT-1292 remove deprecated fields
+  AvailableBalance = 'available-balance',
+}
+
+const getUsageValue = (value: Nullable<TradeState<number>>) => {
+  const currentValue = value?.current;
+  const postOrderValue = value?.postOrder;
+  const hasDiffPostOrder = postOrderValue !== null && currentValue !== postOrderValue;
+  return (hasDiffPostOrder ? postOrderValue : currentValue) ?? 0;
+};
+
+export const AccountInfoSection = () => {
+  const stringGetter = useStringGetter();
+  const dispatch = useAppDispatch();
+
+  const { isTablet } = useBreakpoints();
```

Comment:
for the tablet version, should we just have them all align?
<img width="501" alt="image" src="https://github.com/user-attachments/assets/fb394f57-9f1e-4965-be82-06d49b82681d">

Reply:
or actually we probably don't even need those entry points since profile is more accessible
(or we should actually open the transfers dialog instead for the deposit/withdraw/transfers)

Reply:
hmm I think we have to rehaul tablet views - cuz this view sorta just feels empty in general; given that it matches existing state, would rather keep it like this - but filed as a [subtask](https://linear.app/dydx/issue/CT-1303/[wallet]-consider-aligning-better-using-the-space) of fixing tablet viewz

Reply:
> we should actually open the transfers dialog instead for the deposit/withdraw/transfers 

hmm what do u mean by this? the current state has deposit / withdraw / transfer - did you want just a single entry point?
<img width="904" alt="Screenshot 2024-10-21 at 12 28 11 PM" src="https://github.com/user-attachments/assets/14585f64-7c2c-4766-8889-b3b7b68fb250">